topics/ecology/tutorials/phylogeny-data-prep/tutorial.md

```diff
@@ -0,0 +1,376 @@
+RepeatMasker will only accept compact fasta headers. Before we can mask repetitive regions with RepeatMasker we must trim the NCBI long header (*">BK006939.2 TPA_inf: Saccharomyces cerevisiae S288C chromosome V, complete sequence"*) to leave only the accession number ("*>BK006939.2*") by using a regular expression.
+> ### {% icon hands_on %} Hands-on: Replace Text
+>
+> 1. {% tool [Replace Text](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_replace_in_line/1.1.2) %} with the following parameters:
+>    - {% icon param-collection %} *"File to process"*: `output` (Input dataset collection)
+>    - In *"Replacement"*:
+>        - {% icon param-repeat %} *"Insert Replacement"*
+>            - *"Find pattern"*: `(>[^ ]+).+`
+>            - *"Replace with:"*: `\1`
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    >`\1` replaces the text in the header with the first matched text in the header, the accession number in this case.
+>    {: .comment}
+>
+{: .hands_on}
+
+
+> ### {% icon hands_on %} Hands-on: Mask repetitive sequences
+>
+> 1. {% tool [RepeatMasker](toolshed.g2.bx.psu.edu/repos/bgruening/repeat_masker/repeatmasker_wrapper/4.1.2-p1+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Genomic DNA"*: `outfile` (output of **Replace Text** {% icon tool %})
+>    - *"Repeat library source"*: `DFam (curated only, bundled with RepeatMasker)`
+>        - *"Select species name from a list?"*: `No`
+>            - *"Repeat source species"*: `"Saccharomyces cerevisiae"`
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > In principal, all unique clade names occurring in [NCBI taxonomy database](https://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html) can be used for species. Capitalization is ignored, multiple words need to bound by apostrophes. Not all "common" English names occur in the taxonomy database. Using Latin names is always safest.
+>    {: .comment}
+>
+{: .hands_on}
+
+## Annotate with Funannotate
+We will predict protein-coding genes from genomic sequences using [Funannotate](https://funannotate.readthedocs.io/) ({% cite Young2019 %}), which collects evidence from different ab-initio gene predictors as well as from RNA-seq or ESTs data. Funannotate has been developed for Fungi but it works with any Eukaryotic genome. The output of Funannotate is a list of ORFs and their translation in GenBank format.
+
+If you would like to learn about genome annotation in more depth, the GTN has a [section]({{ site.baseurl }}/topics/genome-annotation) dedicated to training on genome annotation, including a hands-on tutorial on [Funannotate]({{ site.baseurl }}/topics/genome-annotation/tutorials/funannotate/tutorial.html) ({%cite genome-annotation-funannotate %}).
+
+**Even for a small dataset, Funannotate can take a very long time to run. You can skip this step and use the Genbank files downloaded from Zenodo for the following step. These were generated using Funannotate as described in the hands-on below.**
```

Comment:
```suggestion
> ### {% icon warning %} Slow Step Ahead!
> Even for a small dataset, Funannotate can take a very long time to run. You can skip this step and use the Genbank files downloaded from Zenodo for the following step. These were generated using Funannotate as described in the hands-on below.
{: .warning}
```

we can use a nice tip box here instead, and then it's maybe nicer for users to read, and more visibly called out.